modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -89,9 +91,12 @@ export const ListEntityEditor: React.FC<Props> = props => {
   }
 
   const editOccurrence = (idx: number, occurrence: NLU.EntityDefOccurrence) => {
-    const newSynonym = _.last(occurrence.synonyms)
-    if (!isUnique(newSynonym)) {
-      return toastFailure('Synonyms duplication is not allowed')
+    console.log('Occ', occurrence.synonyms)
+    console.log('State', state.occurrences[idx].synonyms)
+    if (occurrence.synonyms.length > state.occurrences[idx].synonyms.length) {
```

Comment:
this is the fix right?

Reply:
The added condition allow to check when adding and by so it bypass when removing (because we're sure to have added only unique synonyms)

The other fix is in the `isUnique` function and bypass the debounce with `[....state.occurence, ...occurences]`